src/Maestro/SubscriptionActorService/PullRequestActor.cs

```diff
@@ -312,9 +312,13 @@ public async Task<ActionResult<bool>> ProcessPendingUpdatesAsync()
 
             (InProgressPullRequest pr, bool canUpdate) = await SynchronizeInProgressPullRequestAsync();
 
+            var subscriptionIds = updates.Count > 1
+                ? "subscriptions " + string.Join(", ",updates.Select(u => u.SubscriptionId).Distinct())
+                : "subscription " + updates[0].SubscriptionId;
+
             if (pr != null && !canUpdate)
             {
-                _logger.LogInformation($"PR {pr?.Url} cannot be updated.");
+                _logger.LogInformation("PR {url} cannot for {subscriptions} be updated", pr.Url, subscriptionIds);
```

Comment:
The wording is a bit confusing

Maybe `Pr {url} cannot be updated for {subscriptions}` would be better here?

Reply:
Good catch